From 7693ec8b167a1b82b58d997f79640048e4e896b5 Mon Sep 17 00:00:00 2001
From: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
Date: Mon, 14 Dec 2015 07:23:36 -0800
Subject: [PATCH] chipsec: setup: install cores library under helper/linux

Simply setting 'cores' in the constructor of Extension causes the library
to be installed under the root of the package. However, the helper.py module
expects to find it under [package-root]/helper/linux. Thus, install the library
where helper.py can find it.

Signed-off-by: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
---
 source/tool/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/tool/setup.py b/source/tool/setup.py
index 7dce743..66a74d5 100644
--- a/source/tool/setup.py
+++ b/source/tool/setup.py
@@ -49,7 +49,7 @@ if platform.system().lower() == "windows":
 
 if platform.system().lower() == "linux":
     data_files = []
-    extensions = [ Extension('cores', sources=['chipsec/helper/linux/cores.c']) ]
+    extensions = [ Extension('chipsec.helper.linux.cores', sources=['chipsec/helper/linux/cores.c']) ]
 
 version      = ""
 VERSION_FILE = os.path.join( os.path.dirname( __file__ ),'VERSION' )
-- 
1.9.1

